routes/api.js

```diff
@@ -124,4 +124,45 @@ router.post('/login', passport.authenticate('local'), (req, res) => {
   res.json(req.user);
 });
 
+router.get('/users', (req, res) => {
+  let dbFilter = {};
+  if (req.query.firstName) {
```

Comment:
It might be a better idea to be able to filter by username instead of firstName if you only have one filter support.